frontend/src/themes/typography.ts

```diff
@@ -60,6 +60,12 @@ const textStyles = {
     lineHeight: "18px",
     letterSpacing: "-0.02em",
   },
+  mobileSubtitle1: {
```

Comment:
what is `mobileSubtitle1`?

Reply:
It's from the [Jump Math Design System](https://www.figma.com/file/Uia7C5xwqDlkLnrXMyt5hn/Jump-Math-Design-System?node-id=12%3A334&t=huzN42tabysdwLnL-0); @vyomeshiyengar said it's the typography used for error states.

Reply:
ooo sg! sorry i vyo and i had a miscommunication here.